### ISSUE TITLE
examples to continue pretrain ViT and other variants like BEiT?

### ISSUE THREAD

**Issue by RobertHua96** [NielsRogge/Transformers-Tutorials#177]
Thanks for the great fine tuning tutorials @NielsRogge! I'm wondering if it would be possible for us to get an example on how we could continue to pre train ViT / BEiT with MIM?

I'm not sure if we could use trainer API for that - and how best to construct a DataLoder / DataCollator for the task.

**Comment by NielsRogge** [comment]
Hi,

Thank you :) we provide a script (`run_mim.py`) for masked image modeling here -> https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-pretraining.

This enables you to both pre-train from scratch or further pre-train models.
